scripts/formatting

```diff
@@ -31,9 +31,6 @@ def run(args: typing.Sequence[typing.Union[str, pathlib.Path]]) -> bool:
 def run_tools(*, check: bool) -> str:
     langs = []
 
-    if not run(('cargo', 'fmt') + (('--', '--check') if check else ())):
-        langs.append('Rust')
-
```

Comment:
What's this change?

Reply:
As with `clippy` ensuring Rust formatting has moved to `nextest` in a previous PR.